docs/topics/wasm/wasm-debugging.md

```diff
@@ -0,0 +1,149 @@
+application built with Kotlin/Wasm.
+
+## Before you start
+
+Create a project using the Kotlin Multiplatform wizard:
+
+1. Open the [Kotlin Multiplatform wizard](https://kmp.jetbrains.com/#newProject).
+2. On the **New Project** tab, you can change the project name to "WasmDemo" and the project ID to "wasm.project.demo".
+3. Select the **Web** option.
+4. Click the **Download** button and unpack the resulting archive.
+
+![Kotlin Multiplatform wizard](wasm-compose-wizard.png){width=600}
+
+## Open the project in IntelliJ IDEA
+
+1. Download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/).
+2. On the Welcome screen of IntelliJ IDEA, click **Open** or select **File | Open** in the menu bar.
+3. Navigate to the unpacked "WasmDemo" folder and click **Open**.
+
+## Run the application
+
+1. In IntelliJ IDEA, open the **Gradle** tool window by selecting **View** | **Tool Windows** | **Gradle**.
+
+   > You need at least Java 11 as your Gradle JVM for the tasks to load successfully.
+   >
+   {type="note"}
+
+2. In **composeApp** | **Tasks** | **kotlin browser**, select and run the **wasmJsBrowserRun** task.
+
+   ![Run the Gradle task](wasm-gradle-task-window.png){width=600}
+
+   Alternatively, you can run the following command in the terminal from the `WasmDemo` root directory:
+
+   ```bash
+   ./gradlew wasmJsBrowserRun
+   ```
+
+3. Once the application starts, open the following URL in your browser:
+
+   ```bash
+   http://localhost:8080/
+   ```
+
+   >The port number can vary because the 8080 port may be unavailable. You can find the actual port number printed
+   > in the Gradle build console.
+   >
+   {type="tip"}
+
+   You can see a "Click me!" button. Click it:
+
+   ![Click me](wasm-composeapp-browser-clickme.png){width=650}
+
+   Now you can see the Compose Multiplatform logo:
+
+   ![Compose app in browser](wasm-composeapp-browser.png){width=650}
+
+## Debug in the browser
+
+> Currently, debugging is only available in the browser. Debugging in 
+> [IntelliJ IDEA](https://youtrack.jetbrains.com/issue/KT-64683/Kotlin-Wasm-debugging-in-IntelliJ-IDEA) and 
+> [Fleet](https://youtrack.jetbrains.com/issue/KT-64684) is in the plan. 
+>
+{type="note"}
+
+You can debug this Compose Multiplatform application
+in your browser out of the box without additional configurations. 
+
+However, for other projects, you may need to set the required configuration in the Gradle 
+build file. For more information, see [Configuration for debugging](#configuration-for-debugging).
+
+> For this tutorial, we are using Chrome. If you use a different browser, the experience should be similar and the following 
+> steps should also work. 
+> 
+{type="tip"}
+
+To debug a Kotlin/Wasm application:
+
+1. In the browser window of the application, access developer tools by right-clicking and selecting the **Inspect** action.
```

Comment:
Can you change the construction of your instructions for this section so instead of: \<purpose of action\> **by** \<instruction\>
So that you have the instruction first?
For example:
> In the application's browser window, right-click and select 'Inspect' to access the developer tools.

This style is more direct and active.